client/my-sites/scan/main.tsx

```diff
@@ -155,20 +155,28 @@ class ScanPage extends Component< Props > {
 			<>
 				<SecurityIcon icon="in-progress" />
 				{ this.renderHeader( heading ) }
+				<p className="scan__progress-bar-percent">{ scanProgress }%</p>
 				<ProgressBar value={ scanProgress } total={ 100 } color="#069E08" />
 				{ isInitialScan && (
 					<p>
```

Comment:
In the design, this whole text is between the heading and the progress bar. Also, it is sized at 24px which does not seem to be the case here currently.

Reply:
> Also, it is sized at 24px which does not seem to be the case here currently.
@miguelpeixe Wanna make it 24px, too? Not sure how important that is 🤷🏻‍♂️

Reply:
It's already 24px, the $font-title-medium is defined as 24px [here](https://github.com/Automattic/wp-calypso/blob/trunk/packages/typography/styles/variables.scss#L42).

Reply:
Hmm, looking good now - I must have misclicked somewhere, sorry about that 😊